tests.py

```diff
@@ -337,6 +337,25 @@ def test_as_var(self):
         self.static.assert_called_with('static.png')
 
 
+class DjangoNowTest(SimpleTestCase):
+    def setUp(self):
+        self.env = Environment(extensions=[DjangoNow])
+    
+    def test_simple(self):
+        template = self.env.from_string("{% now 'Y' %}")
+        expected = timezone.now().strftime('%Y')
```

Comment:
I think, it's better to mock `timezone.now()` here so that the test doesn't fail if something is wrong with the system time.